themes-default/slim/src/api.js

```diff
@@ -8,7 +8,7 @@ const apiKey = document.body.getAttribute('api-key');
  */
 const apiRoute = axios.create({ // eslint-disable-line no-unused-vars
     baseURL: webRoot + '/',
-    timeout: 10000,
+    timeout: 30000,
```

Comment:
The timeouts are needed for shows with 20+ seasons.

Waiting on https://github.com/pymedusa/Medusa/pull/4970

Reply:
That PR won't change anything about the time it takes to generate the data... Just saying.

Reply:
It seems to be directly related to multiple requests. If I move the request for the detailed show to a domain(bypassing the x request per domain/ip limit) I don't see this issue.